test/functional/p2p_segwit.py

```diff
@@ -492,42 +551,42 @@ def test_witness_block_size(self):
         # The witness program will be a bunch of OP_2DROP's, followed by OP_TRUE.
         # This should give us plenty of room to tweak the spending tx's
         # virtual size.
-        NUM_DROPS = 200 # 201 max ops per script!
-        NUM_OUTPUTS = 50
+        num_drops = 200  # 201 max ops per script!
+        num_outputs = 50
```

Comment:
Imo we indicate "compile time" constants with upper case, so I'd prefer if they were left that way, since it makes it easier to read.

Reply:
C.f. https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style

> Constant names are all uppercase, and use _ to separate words.

Reply:
That's not fair! Those are C++ style guides.

But fine, reverted to all caps.